icons/elasticcloud.svg

```diff
@@ -1 +1 @@
-<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Elastic Cloud icon</title><path d="M22.62 19.46a11.95 11.95 0 0 1-9.32 4.48 11.95 11.95 0 0 1-9.32-4.48A11.95 11.95 0 0 1 13.3 15c3.76 0 7.14 1.76 9.32 4.48zM10.32 12c0-1.52 1.25-2.99 2.62-2.99h.42c3.91 0 7.1-1.76 9.29-4.48A12.05 12.05 0 0 0 13.3.05a11.95 11.95 0 0 0-9.32 4.48A11.83 11.83 0 0 0 1.35 12c0 1.94.45 3.76 1.29 5.38a14.3 14.3 0 0 1 7.85-4.45c-.09-.3-.18-.63-.18-.93z"/></svg>
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Elastic Cloud icon</title><path d="M22.669 19.5a12.005 12.005 0 01-9.363 4.5 12.005 12.005 0 01-9.363-4.5 12.005 12.005 0 019.363-4.481c3.777 0 7.173 1.768 9.363 4.5zm-12.357-7.495c0-1.527 1.256-3.004 2.632-3.004h.422c3.928 0 7.133-1.768 9.333-4.5A12.105 12.105 0 0013.306 0a12.005 12.005 0 00-9.363 4.5 11.884 11.884 0 00-2.642 7.505c0 1.949.452 3.777 1.296 5.405a14.366 14.366 0 017.886-4.47c-.09-.302-.18-.634-.18-.935z"/></svg>
```

Comment:
Seem to be a few big changes to their logo since icon was submitted.
I used `Solution Product Logo Glyphs\SVG\ink\cloud-logo-ink-64px.svg` from within the [logo package](https://static-www.elastic.co/v3/assets/bltefdd0b53724fa2ce/bltf9985b332a514521/5ea8c8963a474c52f7244b16/elastic-solution-product-logos.zip).
Would you mind rebuilding?

![image](https://user-images.githubusercontent.com/1830867/102891885-e97b9a00-4456-11eb-8bcd-f8af2ab34ad8.png)

Reply:
Done